### PR TITLE
Feature/cplp 483 admin registration requests Page and unit tests for page

### PR DESCRIPTION
**Pull request by fatihayyildiz**
RegistrationRequests page header implemented
RegistrationRequests page navigation implemented to user menu
Redux store usage some alignment for page unit tests
Unit test and integration tests implemented for RegistrationRequests page

**Table Shared Component Changes**
Number of columns property removed. Component itself counts rows and set number at title
Toolbar Search icon and Filter icon colours are changed based with design

### CODE REVIEW

**Comment by fatihayyildiz** [comment]
@oyo There is some changes to be able to activate unit test for page. I tried to put some comments as much as I can. We can take a detail look together if you want :)

**Comment by fatihayyildiz** [comment]
@christianraysono I made small changes for Table component. Could you please take a look :) ?

File: cx-portal-shared-components/src/components/basic/Table/components/Toolbar/index.tsx [hunk 1]
```diff
@@ -89,18 +89,16 @@ export const Toolbar = ({
...
-            )}
+            <Box
+              component="span"
+              sx={{
+                typography: 'body1',
+                color: 'text.tertiary',
+                marginLeft: 0.5,
+              }}
+            >
+              ({rowCount})
```
Comment (christianraysono) [inline cx-portal-shared-components/src/components/basic/Table/components/Toolbar/index.tsx:100]: Thanks for renaming!

**Review by christianraysono — APPROVED** [review]
Table looks good

**Review by DianaRenz — APPROVED** [review]

**Review by floriansommerraysono — APPROVED** [review]

**Review by oyo — CHANGES_REQUESTED** [review]
i am getting a lot of errors/warnings trying to build the library on this branch:

(!) Plugin typescript: @rollup/plugin-typescript TS2582: Cannot find name 'describe'. Do you need to install type definitions for a test runner? Try `npm i --save-dev @types/jest` or `npm i --save-dev @types/mocha`.
src/components/basic/Table/components/Toolbar/helper.test.ts: (23:1)

23 describe('TableToolbarTest', () => {
   ~~~~~~~~

**Review by oyo — APPROVED** [review]
added commit to get rid of the error messages - now approved :)

**Comment by fatihayyildiz** [comment]
Merge conflict resolved. I am merging to main branch

**Pull request merged**